tq42/dataset.py

```diff
@@ -96,8 +95,8 @@ def list_all(client: TQ42Client, project_id: str) -> List[Dataset]:
 
     For details, see (TODO: update link once a new documentation URL is created)
     """
-    list_datasets_request = ListDatasetsRequest(project_id=project_id)
-    res: ListDatasetsResponse = client.dataset_client.ListDatasets(
+    list_datasets_request = ListStoragesRequest(project_id=project_id)
```

Comment:
if you want to have datasets instead of storages shouldn't you then also filter by type? 🤔